incubator/mysqlha/values.yaml

```diff
@@ -34,10 +34,17 @@ mysqlha:
 ##
 persistence:
   enabled: true
-  ## If defined, volume.beta.kubernetes.io/storage-class: <storageClass>
+  ## If defined, storageClassName: <storageClass>
+  ## If set to "-", storageClassName: "", which disables dynamic provisioning
+  ## If undefined (the default) or set to null, no storageClassName spec is
+  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+  ##   GKE, AWS & OpenStack)
```

Comment:
But...what about Azure?!

Reply:
I've never used Azure so I don't know what is used there. `azure-disk` perhaps - referring to https://docs.microsoft.com/en-us/azure/aks/azure-disks-dynamic-pv?

Reply:
Added documentation for default provisioner used in Azure.